packages/preact-fela/index.d.ts

```diff
@@ -0,0 +1,2078 @@
+import Preact from "preact";
+import { ConsumerProps } from "preact-context";
```

Comment:
There is no need for `preact-context` anymore. Starting with Preact X we ship with the `createContext` in core :+1:

Reply:
No worries, I actually submitted #760 where I change that.

Reply:
might make sense to do that first